protocol_rfcs/README.md

```diff
@@ -16,12 +16,13 @@ Here is the history of all the RFCs propose/accepted/rejected since Feb 6, 2024,
 
 ### Proposed RFCs
 
-| Date proposed | RFC file                                                                                                         | Github issue                                  | RFC title                     |
-|:--------------|:-----------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:------------------------------|
-| 2023-02-02    | [in-commit-timestamps.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/in-commit-timestamps.md)   | https://github.com/delta-io/delta/issues/2532 | In-Commit Timestamps          |
-| 2023-02-09    | [type-widening.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/widening.md)                      | https://github.com/delta-io/delta/issues/2623 | Type Widening                 |
-| 2023-02-14    | [managed-commits.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/managed-commits.md)             | https://github.com/delta-io/delta/issues/2598 | Managed Commits               |
-| 2023-02-28    | [vacuum-protocol-check.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/vacuum-protocol-check.md) | https://github.com/delta-io/delta/issues/2630 | Enforce Vacuum Protocol Check |
+| Date proposed | RFC file                                                                                                                          | Github issue                                  | RFC title                     |
+|:--------------|:----------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------|:------------------------------|
+| 2023-02-02    | [in-commit-timestamps.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/in-commit-timestamps.md)                    | https://github.com/delta-io/delta/issues/2532 | In-Commit Timestamps          |
+| 2023-02-09    | [type-widening.md](https://github.com/delta-io/delta/blob/master/protocol_rfcs/widening.md)                                       | https://github.com/delta-io/delta/issues/2623 | Type Widening                 |
```

Comment:
If you don't mind, could you include this small fix? https://github.com/delta-io/delta/pull/2694